sdcm/remote.py

```diff
@@ -963,3 +971,429 @@ def run_output_check(self, cmd, timeout=None, ignore_status=False,
 
         if not ignore_status and result.exit_status > 0:
             raise process.CmdError(command=result.command, result=result)
+
+
+class RemoteFabric(object):
+
+    def __init__(self, hostname, user="root", port=22, password="",
+                 key_file=None, wait_key_installed=0, extra_ssh_options=""):
+
+        self.hostname = hostname
+        self.key_file = key_file
+        self.user = user
+        self.port = port
+        self.password = password
+        self.extra_ssh_options = extra_ssh_options
+        logger = logging.getLogger('avocado.test')
+        self.log = SDCMAdapter(logger, extra={'prefix': str(self)})
+
+        self.connect_timeout = 60
+        self._use_rsync = False
+        self.known_hosts_file = tempfile.mkstemp()[1]
+        config = Config(overrides={
+                        'load_ssh_config': False,
+                        'UserKnownHostsFile': self.known_hosts_file,
+                        'ServerAliveInterval': 300,
+                        'StrictHostKeyChecking': 'no'})
+        self.connection = Connection(
+            self.hostname,
+            self.user,
+            port=self.port,
+            config=config,
+            connect_timeout=self.connect_timeout,
+            connect_kwargs={
+                'key_filename': os.path.expanduser(self.key_file),
+            })
+
+    def __str__(self):
+        return 'Remote [%s@%s]' % (self.user, self.hostname)
```

Comment:
Remote or RemoteFabric?

Reply:
Agree, for transition period it is better to set RemoteFabric